### ISSUE TITLE
Document environment variables

### ISSUE THREAD

**Issue by rbroth** [micronutrientsupport/api#233]
The readme doesn't make it clear that you have to create a .env file, nor what the different variables are

**Comment by bgsandan** [comment]
True.  In the immediate short term, refer to [.env.example](https://github.com/micronutrientsupport/api/blob/develop/.env.example) and [.env.defaults](https://github.com/micronutrientsupport/api/blob/develop/.env.defaults) but yes, better documentation is needed

**Comment by rbroth** [comment]
I am, and the db names are relatively self-explanatory, but not sure what (if anything) to put for the github and redis variables

**Comment by bgsandan** [comment]
If you aren't testing the `/feedback` endpoints they can probably be blank/empty strings

**Comment by bgsandan** [comment]
At some point, will probably split that bit off from the core data api
